db/schema.rb

```diff
@@ -0,0 +1,64 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_04_30_190600) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comment", force: :cascade do |t|
+    t.bigint "author_id"
+    t.bigint "post_id"
+    t.string "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_comment_on_author_id"
+    t.index ["post_id"], name: "index_comment_on_post_id"
+  end
+
+  create_table "like", force: :cascade do |t|
+    t.bigint "author_id"
+    t.bigint "post_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_like_on_author_id"
+    t.index ["post_id"], name: "index_like_on_post_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.bigint "author_id"
+    t.string "title"
+    t.string "text"
+    t.string "content"
```

Comment:
- [ ] The `text` column should have the type of `text`, not `string`. The type `text` allows for longer storage than `string`. Kindly fix that.

- [ ] I don't think we need an extra column here for `content`. I think the `text` column takes care of whatever `content` is intended for. Kindly remove it to avoid creating unnecessary table columns, or provide some context why it is necessary.